samples/csharp/getting-started/BinaryClassification_CreditCardFraudDetection/CreditCardFraudDetection.Trainer/Program.cs

```diff
@@ -21,8 +21,8 @@ static void Main(string[] args)
             string assetsPath = GetAbsolutePath(AssetsRelativePath);
             string zipDataSet = Path.Combine(assetsPath, "input", "creditcardfraud-dataset.zip");
             string fullDataSetFilePath = Path.Combine(assetsPath, "input", "creditcard.csv");
-            string trainDataSetFilePath = Path.Combine(assetsPath, "output", "testData.csv");
-            string testDataSetFilePath = Path.Combine(assetsPath, "output", "trainData.csv");
+            string trainDataSetFilePath = Path.Combine(assetsPath, "output", "trainData.csv"); 
```

Comment:
Thanks for identifying this. But better if we separate c# and f# changes into different PR's.

I fixed it in master.